integration-tests/environments/react-native/Gemfile.lock

```diff
@@ -91,10 +91,10 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
-  cocoapods (~> 1.11, >= 1.11.2)
+  cocoapods (~> 1.11, >= 1.11.3)
 
 RUBY VERSION
-   ruby 2.7.5p203
+   ruby 2.7.6p219
```

Comment:
This got updated as I ran `bundle install` in the RN test app - I verified the change that triggered this, was already committed separately.

Reply:
Probably could just add this to the gitignore.  Otherwise I'm gonna have to keep downloading new version of ruby when I want to run the integration tests